src/Text/Pandoc/App/CommandLineOptions.hs

```diff
@@ -1098,6 +1175,13 @@ readMetaValue s
   | s == "FALSE" = MetaBool False
   | otherwise    = MetaString $ T.pack s
 
+readBoolFromOptArg :: Maybe String -> Maybe Bool
+readBoolFromOptArg arg = case arg of 
+   Nothing      -> return True
+   Just "true"  -> return True
+   Just "false" -> return False
+   _            -> Nothing
+
```

Comment:
Do you think it would make sense to do `case map toLower arg of` so that `TRUE` and `True` will work also?